docs/changelog.rst

```diff
@@ -1,5 +1,8 @@
 Changelog
 ~~~~~~~~~
+0.40.0 (June 2023)(unreleased)
+------------------------------
+* Added support for the {X, SX, Rz, ECR} in the default compilation pass for :py:class:`IBMQBackend` and :py:class:`IBMQEmulatorBackend`. This is the set of gates used by some of the new IBM devices.
```

Comment:
Can you please combine this to one 0.40. block with the notes from the RC

Reply:
Done. I've combined these under the heading `0.40.0(unreleased)` as the rebase will belong to a different release candidate than the previous changes. Therefore I think this heading should be used instead of RC1.